src/modules/categories/components/CategoryList.js

```diff
@@ -9,9 +9,16 @@ import styled from 'styled-components/native'
 import Image from '../../common/components/Image'
 import CategoryListCaption from '../../../modules/common/components/CategoryListCaption'
 
+export type SimpleModelType = {|
```

Comment:
Maybe `ListElementModelType` is a better name as there is no complex or advanced ModelType

Reply:
Kind of... its a simpler CategoryType with additional `contentWithoutHtml`, but I'd also vote for something different. I don't think `Element` and `Model` are necessary, maybe simply `CategoryListModelType?